src/query/storages/system/src/query_log_table.rs

```diff
@@ -25,6 +29,8 @@ use databend_common_expression::TableSchemaRefExt;
 use serde::Serialize;
 use serde::Serializer;
 use serde_repr::Serialize_repr;
+use sha2::Digest;
+use sha2::Sha256;
```

Comment:
No need sha256, it's too long, md5 is ok.

Reply:
Our query cache use sha generate key. Maybe need same ?

Reply:
Done